package.json

```diff
@@ -5,7 +5,9 @@
   "repository": "https://github.com/BrightspaceUI/core.git",
   "main": "index.html",
   "scripts": {
+    "build:icons": "node ./cli/iconBuilder.js",
```

Comment:
I moved the a11y test helper into a `tools` directory.  What are your thoughts on moving the icon builder into there as well?

Reply:
Yeah good call.

Reply:
Ended up leaving it as the linting rules are very different for `cli` tools vs. tools that get imported into the browser. Easier to use a separate `eslintrc.json` in the `cli` directory.